contracts/deployables/erc20/VotesERC20V1.sol

```diff
@@ -100,12 +104,19 @@ contract VotesERC20V1 is
 
     function supportsInterface(
         bytes4 interfaceId
-    ) public view virtual override returns (bool) {
+    )
+        public
+        view
+        virtual
+        override(Version, AccessControlUpgradeable)
+        returns (bool)
+    {
         return
             interfaceId == type(IVotesERC20V1).interfaceId ||
             interfaceId == type(IERC20).interfaceId ||
             interfaceId == type(IERC20Permit).interfaceId ||
             interfaceId == type(IVotes).interfaceId ||
+            interfaceId == type(IVersion).interfaceId ||
```

Comment:
since this contract inherits from Version, we shouldn't need this line right? I think super.supportsInterface(interfaceId) should handle that for us

Reply:
Both parent contracts implement the `supportsInterface` method, which requires using `override(Version, AccessControlUpgradeable)`. Since `AccessControlUpgradeable` is listed last in the inheritance list, the call to `super.supportsInterface` actually resolves to `AccessControlUpgradeable.supportsInterface`.
I have updated this line to be more explicit: `Version.supportsInterface(interfaceId) ||`